client/executor/common/src/sandbox.rs

```diff
@@ -430,6 +430,19 @@ enum BackendContext {
 	Wasmer(WasmerBackend),
 }
 
+#[cfg(feature = "wasmer-sandbox")]
+fn is_target_suit_for_wasmer() -> bool {
```

Comment:
I don't like `suit` here, sorry 😄
`is_suitable` or `does_suit` is more grammatically correct.

Reply:
Done